agent/virus_total_agent.py

```diff
@@ -75,6 +76,8 @@ def process(self, message: msg.Message) -> None:
                 self._process_response(response, target)
 
     def _process_response(self, response: dict[str, Any], target: str | None) -> None:
+        if target is None:
```

Comment:
the target is handled in `get_technical_details` func + is it normal behavior to have target none

Reply:
in the case of a file target, we get the target from the path value from the message, its shouldnt be the case but it could happen